### PR TITLE
Handle Template for webpack4

### PR DESCRIPTION
**Pull request by SinimaWath**
This PR contains a:

- [x] **bugfix**
- [ ] new **feature**
- [ ] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### Motivation / Use-Case

Template in webpack 4 has different place. 

That fix bug in nextjs. https://github.com/vercel/next.js/issues/25276

### CODE REVIEW

**Review by polyakov713 — APPROVED** [review]

**Review by PsychoSanchez — APPROVED** [review]

**Review by ArchieSA — APPROVED** [review]

**Review by alexander-akait — APPROVED** [review]

**Comment by alexander-akait** [comment]
Can you accept CLA?

**Comment by SinimaWath** [comment]
CLA? What does it mean?

**Comment by alexander-akait** [comment]
https://cla.js.foundation/webpack-contrib/mini-css-extract-plugin?pullRequest=759

**Comment by alexander-akait** [comment]
You can found this in our github actions

**Comment by SinimaWath** [comment]
done

**Comment by alexander-akait** [comment]
@SinimaWath Can you use the same email in commit(s)?

File: src/index.js [hunk 1]
```diff
@@ -619,7 +619,7 @@ class MiniCssExtractPlugin {
         }
       });
 
-      const { Template } = webpack;
+      const Template = webpack.Template || webpack.webpack.Template;
```
Comment (alexander-akait) [inline src/index.js:622]: `webpack.webpack` doesn't exist in `webpack` v4 and v5, also we use:
```
const webpack = compiler.webpack
      ? compiler.webpack
      : // eslint-disable-next-line global-require
        require('webpack');
```
and for v4 and v5 we have only `webpack.Template`

**Comment by alexander-akait** [comment]
Anyway thanks for PR

**Comment by alexander-akait** [comment]
Closing due to inactivity. Please test with latest version and feel free to reopen if still regressions. Thanks!

**Pull request closed without merging**